aggsender/aggsender.go

```diff
@@ -282,26 +292,26 @@ func (a *AggSender) sendCertificate(ctx context.Context) (*agglayertypes.Certifi
 		return nil, fmt.Errorf("error building certificate: %w", err)
 	}
 
-	if rateLimitSleepTime := a.rateLimiter.Call("sendCertificate", false); rateLimitSleepTime != nil {
-		a.log.Warnf("rate limit reached , next cert %s can be submitted after %s so sleeping. Rate:%s",
-			certificate.ID(),
-			rateLimitSleepTime.String(), a.rateLimiter.String())
-		time.Sleep(*rateLimitSleepTime)
-	}
-	a.log.Infof("certificate ready to be sent to AggLayer: %s start: %s , end: %s",
-		certificate.Brief(), startEpochStatus.String(), a.epochNotifier.GetEpochStatus().String())
-	metrics.CertificateBuildTime(time.Since(start).Seconds())
-
 	validatorSignature, err := a.callValidator(ctx, certificate)
```

Comment:
WDYT to rename the `callValidator` to `validateCertificate`?
```suggestion
	validatorSignature, err := a.validateCertificate(ctx, certificate)
```

Reply:
I think that this name cames from this [PR](https://github.com/agglayer/aggkit/pull/738) if @goran-ethernal agrees I can renamed it